<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>543</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>543</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pareto Distribution</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PARETO DISTRIBUTION&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>60</y>
      <width>571</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>571</width>
      <height>201</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>571</width>
      <height>201</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt;</string>
     </property>
     <property name="text">
      <string>ENTER DATA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditScale">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Xm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAlpha">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Î±</string>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt;</string>
     </property>
     <property name="text">
      <string>Scale parameter:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPlot">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>PLOT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt;</string>
     </property>
     <property name="text">
      <string>Shape parameter:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label4_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt;</string>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditX">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>351</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>270</y>
      <width>571</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label1_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt;</string>
     </property>
     <property name="text">
      <string>COMPUTATIONS</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>504</width>
       <height>207</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label5">
          <property name="maximumSize">
           <size>
            <width>63</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string>Median:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelMedian">
          <property name="minimumSize">
           <size>
            <width>431</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>431</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label4">
          <property name="minimumSize">
           <size>
            <width>53</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>53</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string>Mean:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelMean">
          <property name="minimumSize">
           <size>
            <width>441</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>441</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label7">
          <property name="maximumSize">
           <size>
            <width>83</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string>Skewness:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelSkewness">
          <property name="minimumSize">
           <size>
            <width>411</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>411</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label8">
          <property name="maximumSize">
           <size>
            <width>73</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string>Kurtosis:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelKurtosis">
          <property name="minimumSize">
           <size>
            <width>421</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>421</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label6">
          <property name="maximumSize">
           <size>
            <width>73</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string>Variance:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelVariance">
          <property name="minimumSize">
           <size>
            <width>421</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>421</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="label9">
          <property name="minimumSize">
           <size>
            <width>53</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>53</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string>F(x):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelFX">
          <property name="minimumSize">
           <size>
            <width>441</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>441</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
